cmd/gateway-main.go

```diff
@@ -249,9 +249,6 @@ func StartGateway(ctx *cli.Context, gw Gateway) {
 		logger.FatalIf(err, "Unable to initialize disk caching")
 	}
 
-	// Load logger subsystem
-	loadLoggers()
-
 	// Re-enable logging
```

Comment:
Why is `loadloggers` removed for gateway?

Reply:
Its not removed its removed to be called twice @kannappanr